faunadb/faunadb_test.go

```diff
@@ -44,6 +47,15 @@ func SetupTestDB() (client *FaunaClient, err error) {
 		faunaSecret,
 		Endpoint(faunaEndpoint),
 	)
+	if allQueriesTimeout != "" {
+		if millis, err := strconv.ParseUint(allQueriesTimeout, 10, 64); err == nil {
```

Comment:
Could you panic on error here? Otherwise we won't know if we try an invalid config.